src/components/map/MapService.js

```diff
@@ -820,6 +839,37 @@ goog.require('ga_urlutils_service');
           lastLangUsed = lang;
           var url = getLayersConfigUrl(lang);
           return $http.get(url).then(function(response) {
+            // Live modfiications for 3d test
+            if (response.data) {
+              var ids = [
+                'ch.swisstopo.swisstlm3d-karte-farbe',
+                'ch.swisstopo.swissimage-product'
+              ];
+              angular.forEach(ids, function(id) {
+                if (response.data[id]) {
+                  response.data[id].config3d = id + '_3d';
```

Comment:
it depends. I thought, the id can refers to an aggregate layer instead of a list of ids